tests/cfgv_test.py

```diff
@@ -655,3 +657,16 @@ def test_no_additional_keys():
     _assert_exception_trace(excinfo.value, expected)
 
     validate({True: True}, no_additional_keys)
+
+
+warn_additional_keys = Map(
+    'Map', None,
+    Required(True, check_bool),
+    WarnAdditionalKeys((True,), logging),
```

Comment:
once it's changed to a callback, can just use a `mock.Mock()` here

Reply:
Thanks, +1